pkg/kubelet/util/format.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
```

Comment:
pkg/util cannot import pkg/types/api:
"Package util may not depend on any other package in the Kubernetes"

I could move it to pkg/api, but I'm less inclined to do that as these functions are not general enough. I think the first step is to migrate all kubelet logging messages to use this new package (so that they are consistent), then we can consider moving this code to a more general place, and/or merging with other code. For example, kubectl use NamespacedType for printing, which is a bit cumbersome to use in a one-line logging message.